From e17adf9987ed40d8ce665ac5e1339bb49fa299dc Mon Sep 17 00:00:00 2001
From: Pier Luigi Fiorini <pierluigi.fiorini@gmail.com>
Date: Tue, 21 May 2013 23:09:43 +0200
Subject: [PATCH 2/4] Add .config file

Imported from Fedora, but switched TLS backend to be GNUTLS, to match
OpenEmbedded and added a couple of options required by connman.
---
 .gitignore             |    1 +
 wpa_supplicant/.config |   37 +++++++++++++++++++++++++++++++++++++
 2 files changed, 38 insertions(+)
 create mode 100644 wpa_supplicant/.config

diff --git a/.gitignore b/.gitignore
index 7420b72..1630d77 100644
--- a/.gitignore
+++ b/.gitignore
@@ -16,6 +16,7 @@ wpa_supplicant/wpa_gui/Makefile
 wpa_supplicant/wpa_gui/wpa_gui
 wpa_supplicant/wpa_gui-qt4/Makefile
 wpa_supplicant/wpa_gui-qt4/wpa_gui
+!wpa_supplicant/.config
 hostapd/hostapd
 hostapd/hostapd_cli
 hostapd/hlr_auc_gw
diff --git a/wpa_supplicant/.config b/wpa_supplicant/.config
new file mode 100644
index 0000000..ae76ce4
--- /dev/null
+++ b/wpa_supplicant/.config
@@ -0,0 +1,37 @@
+CONFIG_CTRL_IFACE=y
+CONFIG_CTRL_IFACE_DBUS=y
+CONFIG_CTRL_IFACE_DBUS_NEW=y
+CONFIG_CTRL_IFACE_DBUS_INTRO=y
+CONFIG_DRIVER_WEXT=y
+CONFIG_LIBNL32=y
+CONFIG_DRIVER_NL80211=y
+CONFIG_DRIVER_WIRED=y
+CONFIG_IEEE8021X_EAPOL=y
+CONFIG_EAP_MD5=y
+CONFIG_EAP_MSCHAPV2=y
+CONFIG_EAP_TLS=y
+CONFIG_EAP_PEAP=y
+CONFIG_EAP_TTLS=y
+CONFIG_EAP_FAST=y
+CONFIG_EAP_GTC=y
+CONFIG_EAP_OTP=y
+CONFIG_EAP_AKA=y
+CONFIG_EAP_PAX=y
+CONFIG_EAP_LEAP=y
+CONFIG_EAP_SAKE=y
+CONFIG_EAP_GPSK=y
+CONFIG_EAP_GPSK_SHA256=y
+CONFIG_EAP_TNC=y
+CONFIG_WPS=y
+CONFIG_EAP_IKEV2=y
+CONFIG_PKCS12=y
+CONFIG_SMARTCARD=y
+CONFIG_DEBUG_FILE=y
+CONFIG_BACKEND=file
+CONFIG_PEERKEY=y
+CONFIG_BGSCAN_SIMPLE=y
+#CONFIG_FIPS=y
+CONFIG_TLS = gnutls
+CONFIG_GNUTLS_EXTRA=y
+CONFIG_AP=y
+CONFIG_AUTOSCAN_EXPONENTIAL=y
-- 
1.7.10.4

